interface/reports/daily_summary_report.php

```diff
@@ -0,0 +1,409 @@
+                                        <td align="right"><?php echo isset($information['paidAmount']) ? number_format($information['paidAmount'], 2) : number_format(0, 2); ?></td>
+                                        <td align="right">
+                                            <?php
+                                            if (isset($information['payments']) || isset($information['paidAmount'])) {
+                                                $dueAmount = number_format(str_replace(",", "", $information['payments']) - str_replace(",", "", $information['paidAmount']), 2);
+                                            } else {
+                                                $dueAmount = number_format(0, 2);
+                                            }
+                                            echo $dueAmount;
+                                            ?>
+                                        </td>
+                                    </tr>
+                                    <?php
+                                    if (count($dailySummaryReport) > 0) { // calculate the total count of the appointments, new patient,visits, payments, paid amount and due amount
+                                        $totalAppointments += $information['appointments'];
+                                        $totalNewRegisterPatient += $information['newPatient'];
+                                        $totalVisits += $information['visits'];
+                                        $totalPayments += str_replace(",", "", $information['payments']);
+                                        $totalPaidAmount += str_replace(",", "", $information['paidAmount']);
+                                        $totalDueAmount += $dueAmount;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    ?>
+                    <!--display total count-->
+                    <tr class="totalrow">
+                        <td>Total</td>
+                        <td>-</td>
+                        <td>-</td>
+                        <td><?php echo $totalAppointments; ?></td>
+                        <td><?php echo $totalNewRegisterPatient; ?></td>
+                        <td><?php echo $totalVisits; ?></td>
+                        <td align="right"><?php echo number_format($totalPayments, 2); ?></td>
+                        <td align="right"><?php echo number_format($totalPaidAmount, 2); ?></td>
+                        <td align="right"><?php echo number_format($totalDueAmount, 2); ?></td>
+                    </tr>
+                    <?php
+                } else { // if there are no records then display message
+                    ?>
+                    <tr>
+                        <td colspan="9" style="text-align:center;font-weight:bold;"> <?php echo xl("There are no record(s) found."); ?></td>
```

Comment:
html escape here by using the xlt() function instead:
``````
<td colspan="9" style="text-align:center;font-weight:bold;"> <?php echo xlt("There are no record(s) found."); ?></td>